<launch>
	<arg name="agent_id" default="1" />
	
	<node pkg="sunray_swarm" name="rmtt_follow_ugv_$(arg agent_id)" type="rmtt_follow_ugv" output="screen">
		<param name="agent_id" value="$(arg agent_id)" />
		<param name="target_name" value="ugv_1" />
	</node>
</launch>